tests/validation_tests/test_tf_r4_specific_validations.py

```diff
@@ -192,3 +197,101 @@ def test_validate_project_admin_gis_provided_returns_no_failure():
     failures = validate_project_admin_gis_provided(workbook)
 
     assert failures is None
+
+
+def test_validate_funding_profiles_funding_source_failure():
+    funding_df = pd.DataFrame(
+        data=[
+            # Pre-defined Funding Source
+            {
+                "Project ID": "TD-ABC-01",
+                "Funding Source Type": "Towns Fund",
+                "Funding Source Name": PRE_DEFINED_FUNDING_SOURCES[0],
+            },
+            # Invalid "Other Funding Source"
+            {
+                "Project ID": "TD-ABC-01",
+                "Funding Source Type": "Invalid Funding Source Type",
+                "Funding Source Name": "Some Other Funding Source",
+            },
+        ]
+    )
+    workbook = {"Funding": funding_df}
+
+    failures = validate_funding_profiles_funding_source(workbook)
+
+    assert failures == [
+        TownsFundRoundFourValidationFailure(
+            tab="Funding Profiles",
+            section="Project Funding Profiles - Project 1",
+            message='For column "Funding Source", you have entered "Invalid Funding Source Type" which isn\'t '
+            "correct. You must select an option from the list provided",
```

Comment:
I'm not entirely sure what I think about "which isn't correct" - it's more that it isn't a valid. I might just be splitting hairs though. What do you think?

Reply:
No, fair point. I've just taken it straight from the document by content design, like the other enum/dropdown messages. 

It states a generic dropdown error message could be:
`You have entered [number] which isn't correct.  You must select an option from the list provided`.

Reply:
I imagine we just go with the guidance provided for now and if design decide they want it written differently after testing etc then we can change down the line.

Reply:
Yeh, I think the validation messages will be tested with users at some point and we can go from there.